tools/itsg33-issue-generator/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
+pytest
+unittest
```

Comment:
Could you pin these to their latest versions when you get a chance.

Reply:
Done!